go.mod

```diff
@@ -61,3 +61,7 @@ require (
 	k8s.io/kube-openapi v0.0.0-20190722073852-5e22f3d471e6 // indirect
 	k8s.io/utils v0.0.0-20190801114015-581e00157fb1 // indirect
 )
+
+// TODO(qingling128): Change git.apache.org to github.com/apache/thrift in go.sum
+// instead once all our dependencies move off git.apache.org.
+replace git.apache.org/thrift.git => github.com/apache/thrift v0.12.0
```

Comment:
What are the dependencies in question, and what does it take to fix this without incurring the TODO debt?

If we need to leave the TODO, please leave a note on how to check that the TODO is ready to be resolved.

I'm probably missing something, because I don't see any `git.apache.org` dependencies in the list above.

Reply:
From what I see, `Azure` and `Opencencus`:

```
grep -r git.apache.org .
./vendor/github.com/Azure/go-autorest/tracing/go.sum:git.apache.org/thrift.git v0.0.0-20180902110319-2566ecd5d999/go.mod h1:fPE2ZNJGynbRyZ4dJvy6G277gSllfV2HJqblrnkyeyg=
./vendor/go.opencensus.io/Gopkg.lock:  name = "git.apache.org/thrift.git"
./vendor/go.opencensus.io/Gopkg.lock:    "git.apache.org/thrift.git/lib/go/thrift",
./vendor/go.opencensus.io/Gopkg.toml:  name = "git.apache.org/thrift.git"
```

I haven't dug further for what it takes to upgrade those two dependencies. I was thinking about unblocking the release first.

Clarified TODO:
```
// TODO(qingling128): Change git.apache.org to github.com/apache/thrift in
// go.sum instead once all our dependencies move off git.apache.org (e.g. grep
// no longer shows git.apache.org in the vendor folder).
```

Reply:
Thank you, that's very helpful!